postgres/assets/configuration/spec.yaml

```diff
@@ -359,10 +359,36 @@ files:
         - name: collection_interval
           description: |
             Set the query sample collection interval (in seconds). Each collection involves a single query to
-            `pg_stat_activity` followed by at most one `EXPLAIN` query per unique normalized query seen.
+            `pg_stat_activity` followed by at most one `EXPLAIN` query per unique normalized query seen if 
+            collect_explains is activated.
           value:
             type: number
             example: 1
+        - name: samples_per_hour_per_query
+          description: |
+            Set the rate limit for how many query sample events will be ingested per hour per normalized execution
+            plan.
+          value:
+            type: integer
+            example: 15
+        - name: seen_samples_cache_maxsize
+          description: |
+            Set the max size of the cache used for the samples_per_hour_per_query rate limit. This should be increased
+            for databases with a very large number of unique normalized execution plans which exceed the cache's limit.
+          value:
+            type: integer
+            example: 10000
+    - name: explain_plans
```

Comment:
why do we need this new section? we should be able to set `query_samples` -> `enabled: false` & then keep `query_activity` -> `enabled: true`, instead of adding this extra section

Reply:
So I went back and forth on creating a new section.  

The idea here is to have 3 possibilities for users: `query_samples` , `explain_plans`,  and `query_activity`.   Now I could have misunderstood something, but samples for me are just the queries themselves and in the use case presented in the support ticket, the user was most interested in just disabling the explain plans but wanted both activity metrics which requires query_samples to be enabled (probably because of the use of pg_stat_activity no?).

As to the new section, again, I went back and forth about creating a new section for `explain_plans` or leaving this as an option of `query_samples`.  Ultimately I could see this either way, but I just though it might be a tad bit better to have all of the options grouped together. Seeing some of your other comments, though, I think that maybe it's better to just add a new option instead of breaking things out separately.

What do you think on this?